.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "litex-conda-ci"]
```

Comment:
Maybe it would be better to put this as `.travis`?

Reply:
Then it would be probably better to move all files in 'litex-conda-ci' to main dir to avoid `.travis/.travis/*` paths. Should I prepare it as PR?

Reply:
Makes sense to me.

Reply:
Changed. This will be ready to merge after https://github.com/litex-hub/litex-conda-ci/pull/3 is merged.